src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -244,6 +244,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <IlcArg Condition="'$(ExportsFile)' != ''" Include="--exportsfile:$(ExportsFile)" />
       <IlcArg Condition="'$(_targetOS)' == 'win' and '$(DebuggerSupport)' != 'false'" Include="--export-dynamic-symbol:DotNetRuntimeDebugHeader,DATA" />
       <IlcArg Condition="'$(_targetOS)' != 'win' and '$(DebuggerSupport)' != 'false'" Include="--export-dynamic-symbol:DotNetRuntimeDebugHeader" />
+      <IlcArg Condition="'$(_targetOS)' == 'freebsd' and '$(DebuggerSupport)' != 'false'" Include="--export-dynamic-symbol:__progname;--export-dynamic-symbol:environ" />
```

Comment:
Unlike windows, we can't use comma here, it will emit `__progname,environ;` instead of `__progname;environ;`. We can use `%3B` etc. but this is more readable and produces formatted output (one item per line).

Reply:
`DotNetRuntimeDebugHeader,DATA`  on Windows is one symbol name. Comma is not used as symbol separator on Windows if it is what you meant.

Reply:
Yup, I was misreading it. Thanks!